mods/d2k/rules/vehicles.yaml

```diff
@@ -457,9 +457,8 @@ combat_tank_h:
 	Armament:
 		Weapon: 80mm_H
 	Mobile:
-		Speed: 64
-	RevealsShroud:
-		Range: 7c0
+		Speed: 53
+		ROT: 4
```

Comment:
This changes speed as well. Is this intended or an accident. You didn't mention it in the commit description.